packages/askar/src/wallet/AskarBaseWallet.ts

```diff
@@ -125,7 +126,12 @@ export abstract class AskarBaseWallet implements Wallet {
    * Create a key with an optional seed and keyType.
    * The keypair is also automatically stored in the wallet afterwards
    */
-  public async createKey({ seed, privateKey, keyType }: WalletCreateKeyOptions): Promise<Key> {
+  public async createKey({
+    seed,
+    privateKey,
+    keyType,
+    keyBackend = KeyBackend.Software,
```

Comment:
We don't pass it?

Reply:
Need to wait for the library update :). Wanted to have to PR ready as I was waiting.